src/pages/atomicdex/api/v20-dev/non_fungible_tokens/get_nft_list/index.mdx

```diff
@@ -0,0 +1,322 @@
+export const title = "AtomicDEX: Non Fungible Tokens - Get NFT List";
+export const description =
+  "This document describes all the get_nft_list method AtomicDEX-API provides to get a list of your wallets NFTs";
+
+# Get a list of NFTs {{label : 'get_nft_list', tag : 'API-v2'}}
+
+Returns a list of the NFTs owned by the user, shown in descending order of the `block_number` value (the block height when the amount or owner changed). If the request is for NFTs on more than one chain, this means that the order may not be chronological. In the case of ERC1155 tokens, the `block_number` will update when additional NFTs are received or when all NFTs are withdrawn, but will generally remain the same if only some NFTs are withdrawn.
+
+<Note>
+  Before using this method, you must first call the [update\_nft](/atomicdex/api/v20-dev/non_fungible_tokens/update_nft/)
+  method to populate/refresh the local database.
+</Note>
+
+### Request Parameters
+
+| Parameter           | Type    | Description                                                                                                                                       |
+| ------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| chains              | array   | List of chains to scan for NFTs.                                                                                                                  |
+| max                 | boolean | Optional, defaults to `false`. If `true`, response will return all NFTs without pagination, and will ignore the `limit` and `page_number` values. |
+| limit               | integer | Optional, defaults to `10`. The number of NFTs displayed per page in response.                                                                    |
+| page\_number        | integer | Optional, defaults to `1`. The page offset for items in response.                                                                                 |
+| protect\_from\_spam | boolean | Optional, defaults to `false`. If `true`, any potentially spam link found in collection name, token name, symbol will be censored                 |
+| filters             | object  | Optional. A standard [TokenFilter](/atomicdex/api/v20/#token-filter) object.                                                                      |
+
+### Response Parameters
```

Comment:
we should include in response param description `skipped` and `total` fields, as actual response is this object
```rust
pub struct NftList {
    pub(crate) nfts: Vec<Nft>,
    pub(crate) skipped: usize,
    pub(crate) total: usize,
}
```

Reply:
thanks. Added, and the nested object moved to common structures doc